assets/js/components/BatchEdit/Remove.js

```diff
@@ -27,15 +27,15 @@ export default function BatchEditRemove({
       </UIFormField>
 
       {removeItems.length > 0 && (
-        <div className="content">
-          <ul>
+        <div className="content has-background-danger-light">
+          <ul className="py-4">
             {removeItems.map((item) => {
               const { label, role, term } = splitFacetKey(item);
 
               return (
-                <li key={item} className="has-text-danger">
-                  {`${label} - ${term} - ${role}`}
```

Comment:
@katdivyareddy10 @kdid  Do we want to keep `role` here for Contributor and Subject s?

Reply:
I can remove it if it's not useful.

Reply:
Keep the roles for contributor and subject. (only display the role label (not role id)) but removing:
Contributor 1 Term - contributor1id - (Author) 
Contributor 1- contributor1id -  (Publisher)

are two different things - users need to see the difference.

Reply:
Ok, I guess that means we're keeping role.

Reply:
If there are no other changes, could either of you review this PR?